lds/arm/kernel-link.x

```diff
@@ -192,6 +192,30 @@ SECTIONS
     *(.ARM.exidx.*);
     *(.ARM.extab.*);
   }
+
+  .debug_loc (INFO) : { *(.debug_loc) }
+  .debug_abbrev (INFO) : { *(.debug_abbrev) }
+  .debug_info (INFO) : { *(.debug_info) }
+  .debug_aranges (INFO) : { *(.debug_aranges) }
+  .debug_ranges (INFO) : { *(.debug_ranges) }
+  .debug_str (INFO) : { *(.debug_str) }
+  .debug_pubnames (INFO) : { *(.debug_pubnames) }
+  .debug_pubtypes (INFO) : { *(.debug_pubtypes) }
+  .debug_line (INFO) : { *(.debug_line) }
+  .debug_frame(INFO) : { *(.debug_frame) }
+
+  .llvmbc (INFO) : { *(.llvmbc) }
+  .llvmcmd (INFO) : { *(.llvmcmd) }
+
+  .note.GNU-stack (INFO) : { *(.note.GNU-stack) }
+
+  .ARM.attributes (INFO) : { *(.ARM.attributes) }
+
+  .symtab (INFO) : { *(.symtab) }
+  .shstrtab (INFO) : { *(.shstrtab) }
+  .strtab (INFO) : { *(.strtab) }
+
+  .comment (INFO) : { *(.comment) }
```

Comment:
If I understand correctly, the intent is to capture all input sections and have a place for them in the output sections. But we're making these with INFO so that they are not marked allocatable for the program itself? Or was it that you are trying to ensure these input sections are included in the output section of the resulting linked binary?

Reply:
I didn't really know what to put so I just took what I saw on another allocation. But looking at [this](https://home.cs.colorado.edu/~main/cs1300/doc/gnu/ld_3.html#SEC5), I think most of these should be `NOLOAD`, although I'll need to do some digging to determine what.